### PR TITLE
Feat/civ 6141 reset pin event

### PR DESCRIPTION
**Pull request by TatsyMaslova**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CIV-6141

### Change description ###

reset pin event

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by laperla11 — APPROVED** [review]

**Review by laarmada — APPROVED** [review]

File: bin/utils/process-definition.sh [hunk 1]
```diff
@@ -14,10 +14,10 @@ if [[ ! -e ${definition_output_file} ]]; then
    touch ${definition_output_file}
 fi
 
-docker run --rm --name json2xlsx \
+docker run --rm --name json2xlsx --user=$UID\
```
Comment (laarmada) [inline bin/utils/process-definition.sh:17]: take care about that is not for local setup that parameters?

File: bin/utils/process-definition.sh [hunk 2]
```diff
@@ -14,10 +14,10 @@ if [[ ! -e ${definition_output_file} ]]; then
...
   -v ${definition_input_dir}:/tmp/ccd-definition \
   -v ${definition_output_file}:/tmp/ccd-definition.xlsx \
-  -e CCD_DEF_CASE_SERVICE_BASE_URL=${CCD_DEF_CASE_SERVICE_BASE_URL:-http://docker.for.mac.localhost:4000} \
+  -e CCD_DEF_CASE_SERVICE_BASE_URL=${CCD_DEF_CASE_SERVICE_BASE_URL:-http://${DOCKERFORLINUX}:4000} \
```
Comment (laarmada) [inline bin/utils/process-definition.sh:20]: take care about that is not for local setup that parameters?

File: bin/utils/process-definition.sh (hunk 1 above)
Comment (TatsyMaslova) [reply bin/utils/process-definition.sh:17]: oopsie you are right Sergio will revert the change

**Comment by TatsyMaslova** [comment]
reopen for testing

**Review by portizdev — APPROVED** [review]

**Review by GorilaNaranja — APPROVED** [review]

**Review by neeta-hmcts — APPROVED** [review]

**Pull request merged**